src/buildmanager.h

```diff
@@ -181,7 +181,12 @@ public slots:
 	QStringList deprecatedUserToolCommands, deprecatedUserToolNames;
 	QStringList userToolOrder, userToolDisplayNames;
 	enum Dvi2PngMode { DPM_DVIPNG, DPM_DVIPNG_FOLLOW, DPM_DVIPS_GHOSTSCRIPT, DPM_EMBEDDED_PDF, DPM_LUA_EMBEDDED_PDF, DPM_XE_EMBEDDED_PDF};
+	// following Dvi2PngModes add tight page modifications for to the document preamble preview

```

Comment:
```suggestion
	// following Dvi2PngModes add tight page modifications for the document preamble preview
```

Reply:
@muzimuzhi I think I ment 'add ... to the document preamble for preview''. Ok?

Reply:
Sure. It's "for to" that confused me. So you meant
```suggestion
	// following Dvi2PngModes add tight page modifications to the document preamble preview

```

Reply:
I suggest following Dvi2PngModes add tight page modifications to the document preamble used for preview